### PR TITLE
Edit profile information 

### PR DESCRIPTION
**Pull request by AndreyBartnovskiy**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -3,4 +3,7 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+  
+  validates :first_name, presence: true
+  validates :last_name, presence: true
```
Comment (RasimKhusaenov) [inline app/models/user.rb:8]:
```suggestion
  validates :first_name, :last_name, presence: true
```

File: app/views/devise/registrations/new.html.erb [hunk 2]
```diff
@@ -3,6 +3,16 @@
 <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
   <%= render "devise/shared/error_messages", resource: resource %>
 
+  <div class="field">
+    <%= f.label :first_name %><br />
+    <%= f.text_field :first_name, autofocus: true %>
```
Comment (RasimKhusaenov) [inline app/views/devise/registrations/new.html.erb:8]: If I am not mistaken, autofocus works only for one field on page

File: db/migrate/20230313145326_add_names_to_users.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+class AddNamesToUsers < ActiveRecord::Migration[6.1]
+  def change
+    add_column :users, :first_name, :string
+    add_column :users, :last_name, :string
```
Comment (RasimKhusaenov) [inline db/migrate/20230313145326_add_names_to_users.rb:4]: null: false?

File: app/views/devise/registrations/edit.html.erb [hunk 4]
```diff
@@ -3,9 +3,20 @@
 <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
   <%= render "devise/shared/error_messages", resource: resource %>
 
+
+  <div class="field">
+    <%= f.label :first_name %><br />
```
Comment (RasimKhusaenov) [inline app/views/devise/registrations/edit.html.erb:8]: Do you really need `<br>` here?

Comment (AndreyBartnovskiy) [reply app/views/devise/registrations/edit.html.erb:8]: it redundant, sure

**Review by RasimKhusaenov — APPROVED** [review]

**Pull request merged**